### PR TITLE
Fix login and password in form

### PR DESCRIPTION
**Pull request by GermanBluefox**
With old code the user was always "undefined" because the data was like:
`redirect=&client_id=ZxjqWpsYj3&redirect_uri=%2Ffrontend&state=cool_jazz`

### CODE REVIEW

File: smart-home-provider/frontend/login.html [hunk 1]
```diff
@@ -200,17 +200,19 @@
...
             </paper-input>
 
             <input type="hidden" name="redirect">
             <input type="hidden" name="client_id">
             <input type="hidden" name="redirect_uri">
             <input type="hidden" name="state" />
+            <input type="hidden" name="username">
+            <input type="hidden" name="password" />
 
             <paper-button raised onclick="formSubmit()">Login</paper-button>
```
Comment (Fleker) [inline smart-home-provider/frontend/login.html]: I know this is a nitpick on myself, but I should remove the inline JS handler and put it instead in a `<script>` block.

**Comment by Fleker** [comment]
I'm not a Polymer expert, and I've run the sample a few times without seeing issues. Do you know why the current implementation doesn't work?

**Comment by vibgy** [comment]
I had the same issue and this PR fixes it. LGTM.

**Comment by Fleker** [comment]
This change LGTM, although I'm not yet clear what issue is happening. I open the login page and authenticate without issue. Can you explain why the current implementation is causing a problem?

**Comment by GermanBluefox** [comment]
Your page sends user=undefined, login=undefined and server accepts it and
create a token for this login (btw - another issue).

26 окт. 2017 г. 3:06 пользователь "Nick" <notifications@github.com> написал:

**Comment by Fleker** [comment]
Thanks for the help! I've verified you have signed the CLA in the past.

**Pull request merged**